.env.example

```diff
@@ -1,3 +1,5 @@
 REACT_APP_API_PREFIX=""
 SENTRY_DSN=""
 REACT_APP_GLIFIC_API_PORT=4000
+REACT_APP_APPLICATION_NAME=""
```

Comment:
@rathorevaibhav why do we need an app name?

Reply:
hey @kurund I made these fields editable from env so that if needed, SaaS providers who extend Glific can rename it.

For example, like Tides, we can have App name set to "Tides" instead of "Glific". It's just so that we provide flexibility to extend the framework. If there's nothing set, the default value will come in which is "Glific".